aips/create_dips_job.py

```diff
@@ -186,6 +188,14 @@ def filter_aips(aips, location_uuid):
         if aip["current_location"] != location:
             LOGGER.debug("Skipping AIP (different location): %s", aip["uuid"])
             continue
+        if origin_pipeline_uuid:
+            pipeline = "/api/v2/pipeline/{}/".format(origin_pipeline_uuid)
```

Comment:
It's interesting looking at our data sometimes and seeing the implementation details (`/api/v2/`) we sometimes expose!